src/core/targeting/build-page-targeting.ts

```diff
@@ -69,13 +69,14 @@ const filterValues = (pageTargets: Record<string, unknown>) => {
 	return filtered;
 };
 
-const getLastNavigationType = (): NavigationTimingType | undefined => {
+const lastPerformanceEntryIsNavigationType = (): boolean => {
 	if (!supportsPerformanceAPI()) {
-		return undefined;
+		return false;
 	}
 	const navigationEvents = performance.getEntriesByType('navigation');
 	const lastNavigationEvent = navigationEvents[navigationEvents.length - 1];
-	return lastNavigationEvent?.type;
+	// https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/entryType#navigation
+	return lastNavigationEvent?.entryType === 'navigation';
```

Comment:
Legit refactor, see [comment in related DCR PR](https://github.com/guardian/dotcom-rendering/pull/11470#discussion_r1626245453)